parlai/core/torch_agent.py

```diff
@@ -259,7 +260,7 @@ def add_reply(self, text):
         # update history vecs
         self._update_vecs(text)
 
-    def update_history(self, obs):
+    def update_history(self, obs, temp_history=None):
```

Comment:
so `temp_history` is persistent until the next call to `update_history`, right? maybe we can add a small comment to `update_history` to make that more explicit